CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ include(CheckIncludeFiles)
 include(CheckIncludeFileCXX)
 include(CheckFunctionExists)
 
+#put all the libs and binaries in one place
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
```

Comment:
@jcsp I'm introducing the lib, bin directories thing in another PR i'll have out today, which might take a little longer to get merged. I wanted to keep things consistent when I rebase There are some updates to vstart.sh in that one as well.